.spelling

```diff
@@ -898,10 +898,14 @@ Zhihan
 Zhong
 Zhonghu
 Zipkin
+MB
+MiB
+GB
+GiB
+CPU
+CPUs
+
 
  - search.md
```

Comment:
Remove the whole search.md section. The file doesn't exist anymore so just remove all of it.